templates/default/client.html.twig

```diff
@@ -49,7 +49,7 @@
 
             <div class="panel panel-default">
               <div class="panel-heading panel-collapsable-heading">
-                <a class="collapsed" href="#viewadvanced" data-toggle="collapse" data-target="#viewadvanced">Advanced</a></div>
+                <a class="collapsed" href="#viewadvanced" data-toggle="collapse" data-target="#viewadvanced">{% trans %}Advanced{% endtrans %}</a></div>
```

Comment:
Hi! This correction is not related to the french translation.
Could you use a separate PR for these changes? Thanks!